AsyncDisplayKit/ASEditableTextNode.h

```diff
@@ -19,6 +19,12 @@
 
 #pragma mark - Configuration
 
+/**
+  @abstract Access to underlying UITextView for more configuration options.
+  @warning This property should only be used on the main thread and should not be accessed before the editable text node's view is created.
+ */
+@property (nonatomic, readwrite, strong) UITextView *textView;
```

Comment:
This should not be a readwrite property as we don't want arbitrary UITextViews to be able to assigned here, since it needs to be a custom subclass.